tap_powerbi_metadata/streams.py

```diff
@@ -25,167 +23,6 @@
 
 API_DATE_FORMAT = "'%Y-%m-%dT%H:%M:%SZ'"
```

Comment:
This Line needs to be moved/copied to client.py since it is used on lines 154 and 179  and throwing an error when syncing the ActivityEvents stream

```
An unhandled error occurred while syncing 'ActivityEvents'
...
NameError: name 'API_DATE_FORMAT' is not defined
```